### DISCUSSION TITLE
I need help in github account deletion

### DISCUSSION THREAD

**Discussion by tilak-git** [community/community#64188] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, 
I had a question regarding in account deletion. if i delete my account will it be deleted immediately or i heard that it takes 90 days to get account deleted permanently.

**Comment by nirav-gajera** [comment]
it will not be deleted immediately. It will be archived for 90 days, during which time you can restore it if you change your mind. After 90 days, your account will be permanently deleted.

**Reply by tilak-git** [reply]
Is there any way to delete it permanently before 90 days?

**Reply by nirav-gajera** [reply]
from my knowledge there is no way to delete before 90 days
